browser/help/topics/v3-ht-annotations.md

```diff
@@ -165,22 +165,22 @@ The gnomAD v3 Hail Table annotations are defined below:
+  - **cadd**: Combined Annotation Dependent Deletion [score](https://cadd.gs.washington.edu/) used to predict deleteriousness of SNVs and indels.
+    - **phred**: CADD Phred-like scaled C-scores ranging from 1 to 99 based on the rank of each variant relative to all possible 8.6 billion substitutions in the human reference genome. Larger values indicate increased predicted deleteriousness.
+    - **raw_score**: Unscaled CADD scores indicating whether a variant is likely to be "observed" (negative values) vs "simulated" (positive values). Larger values indicate increased predicted deleteriousness.
+  - **sift**: Sorting Tolerant From Intolerant (SIFT) [score](https://www.nature.com/articles/nprot.2009.86) used to predict the impact of SNVs on protein function.
+    - **sift_max**: Scaled probability of the amino acid substitution being tolerated, ranging from 0 to 1. Scores below 0.05 are predicted to impact protein function.  A normalized probability of observing the new amino acid at that position to affect protein function, and ranges from 0 to 1. We prioritize max scores for MANE Select transcripts where possible and otherwise report a score for the canonical transcript. 
+  - **polyphen**: Polymorphism Phenotyping v2 (Polyphen-v2 [score](https://www.nature.com/articles/nmeth0410-248)) that predicts the impact of an SNV on protein structure and function.
+    - **polyphen_max**: Prediction of the possible impact of an amino acid substitution on the structure and function of a human protein, ranging from 0.0 (tolerated) to 1.0 (deleterious).  We prioritize max scores for MANE Select transcripts where possible and otherwise report a score for the canonical transcript.
+  - **phylop**:Per-base evolutionary constraint (score)[https://pubmed.ncbi.nlm.nih.gov/37104612/].
+    - **phylop**: Base-wise conservation across the 241 placental mammals in the [Zoonomia](https://zoonomiaproject.org/) project. Score ranges from -20 to 9.28, and reflects acceleration (faster evolution than expected under neutral drift, assigned negative scores) as well as conservation (slower than expected evolution, assigned positive scores). 
+  - **splice_ai**: [Score](https://linkinghub.elsevier.com/retrieve/pii/S0092-8674(18)31629-5) that predicts a variant's impact on splicing.
+    - **max_ds**: The maximum delta score across 4 splicing consequences, which reflects the probability of the variant being splice-altering. If a variant was predicted to fall within multiple genes, score is across all relevant genes. Score ranges from 0 to 1. In the SpliceAI paper, a detailed characterization of the delta scores is provided for 0.2 (high recall), 0.5 (recommended), and 0.8 (high precision) cutoffs. 
+  - **pangolin**: [Score](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-022-02664-4) predicting a variant's impact on splicing.
```

Comment:
```suggestion
  - **pangolin**: [Score](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-022-02664-4) that predicts a variant's impact on splicing.
```
might as well make the text consistent between this and spliceAI (since this is such a generic description)